plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -545,10 +545,10 @@ private static RowType toRowType(List<ColumnMetadata> columns)
                             .put("t_bigint", createIntegerColumnStatistics(OptionalLong.of(2345L), OptionalLong.of(6789L), OptionalLong.of(4), OptionalLong.of(7)))
                             .put("t_integer", createIntegerColumnStatistics(OptionalLong.of(234L), OptionalLong.of(678L), OptionalLong.of(5), OptionalLong.of(6)))
                             .put("t_smallint", createIntegerColumnStatistics(OptionalLong.of(23L), OptionalLong.of(65L), OptionalLong.of(7), OptionalLong.of(5)))
-                            .put("t_tinyint", createIntegerColumnStatistics(OptionalLong.of(12), OptionalLong.of(3L), OptionalLong.of(2), OptionalLong.of(3)))
```

Comment:
@losipiuk does it make sense for a follow-up task in future to normalize such incorrect stats? Or does the engine already ignore/correct for such stats.

cc: @sopel

Reply:
Hmmmm, @GaruGaru I don't think logically incorrect stats value should be a problem. If you changed these values just because they were logically inconsistent, let's revert the change.

Reply:
I had to change those values because Glue api reject values that are logically inconsistent ( eg `avg > max`)